indexd/index/driver.py

```diff
@@ -19,6 +19,8 @@ def ids(self,
             start=None,
             size=None,
             urls=None,
+            acl=None,
```

Comment:
:grimacing: weird that acl wasn't here before

Reply:
Yeah, it was an abandoned overdesign I think.